adafruit_mcp3xxx/mcp3004.py

```diff
@@ -58,5 +41,5 @@ class MCP3004(MCP3xxx):
     DIFF_PINS = {(0, 1): P0, (1, 0): P1, (2, 3): P2, (3, 2): P3}
 
     def __init__(self, spi_bus, cs, ref_voltage=3.3):
-        super(MCP3004, self).__init__(spi_bus, cs, ref_voltage=ref_voltage)
+        super().__init__(spi_bus, cs, ref_voltage=ref_voltage)
```

Comment:
Same question about the super calls

Reply:
@dherrada For future reference, please include the entire question if it's not in the same repo during the same review process. I had to dig to find your initial question.

This is fine.

Reply:
@kattni Ok. I'll make sure to do that in the future